docs/api/import_materials.md

```diff
@@ -63,3 +64,138 @@ Content-Type application/json
    ]
 }
 ```
+
+## Import a GEO material:
+
+**Request**:
+
+`POST` `/materials/import/`
+
+Parameters:
+
+Name                           | Type   | Description
+-------------------------------|--------|---
+import_type                    | string | The type of resource being imported. Options include: ["SRA", "GEO", "PROTOCOLS.IO"].
+study_accession                | int    | The accession code for the study represeting the SRP material, in the form "SRP123456".
+organization_id                | int    | The ID of the organization that this material will be owned by.
+grant_id                       | int    | The ID of the grant that this material will be associated with.
+
+*Note:*
+
+- To import a GEO material, ```import_type``` must be set to "GEO".
+- All parameters are required.
+- **[Authorization Protected](authentication.md)**
+- The user making the request must own the given grant and be a part of the given organization.
+
+
+**Response**:
+
+```json
+Content-Type application/json
+200 OK
+
+{
+   "id":9,
+   "category":"DATASET",
+   "url":"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE24528",
+   "pubmed_id":[
+      "21430780"
+   ],
+   "additional_metadata":{
+      "study_id":"GSE24528",
+      "description":"Investigation of expression differences between skin and melanomas from a transgenic BRAFV600E zebrafish model of melanomaThe embryos described in this study are further analyzed in a manuscript submitted for publication by White, et al.",
+      "platform":[
+         "GPL1319"
+      ],
+      "technology":[
+         "Expression profiling by array"
+      ],
+      "num_samples":15
+   },
+   "mta_attachment":"None",
+   "title":[
+      "Expression analysis of zebrafish melanoma and skin from the mitf-BRAFV600E;p53-/- line"
+   ],
+   "needs_irb":false,
+   "needs_abstract":false,
+   "imported":true,
+   "shipping_requirements":"None",
+   "import_source":"GEO",
+   "contact_user":"3e9aaef0-7b49-48c4-9f39-191ce7ab5aec",
+   "organization":16,
+   "organisms":[
+      "DANIO_RERIO"
+   ],
+   "publication_title":"None",
+   "pre_print_doi":"None",
+   "pre_print_title":"None",
+   "citation":"None",
+   "additional_info":"None",
+   "embargo_date":"None",
+   "is_archived":false,
+   "grants":[
+      4
+   ]
+}
+```
+
+## Import a protocols.io material:
+
+**Request**:
+
+`POST` `/materials/import/`
+
+Parameters:
+
+Name                           | Type   | Description
+-------------------------------|--------|---
+import_type                    | string | The type of resource being imported. Options include: ["SRA", "GEO", "PROTOCOLS.IO"].
+study_accession                | int    | The accession code for the study represeting the SRP material, in the form "SRP123456".
```

Comment:
This should be updated to reflect what you provide protocols.io. I think we should maybe change `study_accession` to be something more generic like `accession` or `material_accession` so that it can work for every type of material we ever write an importer for. (Post MVP there may be more than these three.)

Reply:
For protocols.io, you provide a DOI- I just changed it to that.